### PR TITLE
chore: 12.11.0 release

### PR DESCRIPTION
**Pull request by jordanpowell88**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: cli/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+- Fixed an issue where setting `videoCompression` to `0` would cause the video output to be broken. `0` is now treated as false. Addresses [#5191](https://github.com/cypress-io/cypress/issues/5191) and [#24595](https://github.com/cypress-io/cypress/issues/24595).
+- Fixed an issue on the [Debug page](https://on.cypress.io/debug-page) where the passing run status would appear even if the Cypress Cloud organization was over its monthly test result limit. Addresses [#26528](https://github.com/cypress-io/cypress/issues/26528).
+
+**Misc:**
+- Cleaned up our open telemetry dependencies, reducing the size of the open telemetry modules. Addressed in [#26522](https://github.com/cypress-io/cypress/pull/26522).
+
+**Dependency Updates:**
+- Upgraded [`vue`](https://www.npmjs.com/package/vue) from `3.2.31` to `3.2.47`. Addressed in [#26555](https://github.com/cypress-io/cypress/pull/26555).
+
 ## 12.10.1
```
Comment (astone123) [inline cli/CHANGELOG.md]: I think we need to delete this entire 12.10.1 section

File: cli/CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
-## 12.10.1
 
-_Released 04/25/2023 (PENDING)_
+## 12.11.0
```
Comment (astone123) [inline cli/CHANGELOG.md:2]: This needs to be on line 2

**Review by astone123 — CHANGES_REQUESTED** [review]
Need to update the changelog again

File: cli/CHANGELOG.md (hunk 2 above)
Comment (jordanpowell88) [reply cli/CHANGELOG.md:2]: removed in f0247bde3af51943378cc18041631d0a630b92d3

**Review by mjhenkes — APPROVED** [review]

File: cli/CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,22 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
-## 12.10.1
+## 12.11.0
 
-_Released 04/25/2023 (PENDING)_
+_Released 04/25/2023_
 
 **Features:**
-
+- Adds Component Testing support for Angular 16. Addresses [#26044](https://github.com/cypress-io/cypress/issues/26044).
```
Comment (astone123) [inline cli/CHANGELOG.md:8]: There needs to be an empty line break after each heading

Comment (jordanpowell88) [reply cli/CHANGELOG.md:8]: Added in 9a86afe7b40d7d013d969923d03917afac715504

**Review by astone123 — APPROVED** [review]

**Pull request merged**